### PR TITLE
update openstack documentation

### PR DESCRIPTION
**Pull request by zetaab**

### CODE REVIEW

File: docs/tutorial/openstack.md [hunk 1]
```diff
@@ -46,6 +50,8 @@ export KOPS_STATE_STORE=swift://<bucket-name> # where <bucket-name> is the name
 export KOPS_FEATURE_FLAGS="AlphaAllowOpenStack"
 ```
 
+Ps. If your OpenStack does not have Swift, you can use for instance S3 to provision kops for OpenStack. All VFS stores are supported.
```
Comment (drekle) [inline docs/tutorial/openstack.md]: I'd probably word this differently but only personal opinion perhaps.
```
Ps. If your OpenStack does not have Swift, you can use for instance S3 to provision kops for OpenStack. All VFS stores are supported.
```
```
If your OpenStack does not have Swift you can any other VFS store, such as S3.
```

File: docs/tutorial/openstack.md [hunk 2]
```diff
@@ -56,31 +62,25 @@ openstack volume type list
...
   --master-count=3 \
   --node-count=1 \
-  --node-size 2vCPUx8GB \
-  --master-size 2vCPUx8GB \
-  --etcd-storage-type CBS \
-   --api-loadbalancer-type public \
+  --node-size <flavorname> \
+  --master-size <flavorname> \
+  --etcd-storage-type <volumetype> \
+  --api-loadbalancer-type public \
```
Comment (drekle) [inline docs/tutorial/openstack.md]: I think the external network now **must** be specified.

```
  --os-ext-net tenant-floatingip-net
```
We might also document this add:
```
  --os-kubelet-ignore-az=true
```

File: docs/tutorial/openstack.md [hunk 3]
```diff
@@ -46,6 +50,8 @@ export KOPS_STATE_STORE=swift://<bucket-name> # where <bucket-name> is the name
 export KOPS_FEATURE_FLAGS="AlphaAllowOpenStack"
 ```
 
+If your OpenStack does not have Swift you can any other VFS store, such as S3.
```
Comment (drekle) [inline docs/tutorial/openstack.md]: `can use`  Perhaps I typo'd this to you in my previous.

File: docs/tutorial/openstack.md [hunk 4]
```diff
@@ -56,31 +62,28 @@ openstack volume type list
...
-  --etcd-storage-type CBS \
-   --api-loadbalancer-type public \
+  --node-size <flavorname> \
+  --master-size <flavorname> \
+  --etcd-storage-type <volumetype> \
+  --api-loadbalancer-type public \
   --topology private \
   --bastion \
   --ssh-public-key ~/.ssh/id_rsa.pub \
   --networking weave
```
Comment (drekle) [inline docs/tutorial/openstack.md]: missed the escape here `\`

**Comment by justinsb** [comment]
Docs so merging manually while test account is offline.

**Comment by zetaab** [comment]
@justinsb btw how often these protokube image and kops base url thins are rebuilt?

I have compiled these myself
```
export PROTOKUBE_IMAGE=https://kopstest.xxx/.build/dist/images/protokube.tar.gz
export KOPS_BASE_URL=https://kopstest.xxx/.build/dist/
```

but if someone now wants to use openstack, these instructions does not work. These two env variables are needed (and building those before that and upload to somewhere). Could we for instance release new alpha which do have updated protokube etc stuff publicly available?

**Pull request merged**